### PR TITLE
fix: change session start log level

### PR DESCRIPTION
**Pull request by element-code**
**Description**
The session class clutters out-of-the-box configured logs with debug information about its driver initialization on startup, using a `INFO` classification.
This is especially problematic in authenticated webapps as every request will write at least one info message. 
The message itself has a debug purpose and therefore should be classified as `DEBUG`

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

**Comment by michalsn** [comment]
I agree, that using `info` may pollute the logs unnecessarily. I wouldn’t be against treating this as a bug.

Still, we will need a changelog entry.

Side note... Items on the issue description checklist should only be marked if they are accurate - thanks!

**Comment by kenjis** [comment]
I don't think this is a bug, but I would not be against to change to *DEBUG*.
I think it is better that this PR goes to 4.6 branch.

**Comment by element-code** [comment]
Which file I need to edit for the changelog entry  @michalsn?
I think this could be treated as a bug, since it only relates to application debugging and shouldn't be breaking in any production environment, therefore.

**Comment by michalsn** [comment]
@element-code This one: https://github.com/codeigniter4/CodeIgniter4/blob/develop/user_guide_src/source/changelogs/v4.5.6.rst

File: user_guide_src/source/changelogs/v4.5.6.rst
Comment (michalsn) [inline user_guide_src/source/changelogs/v4.5.6.rst:32]:
```suggestion
- **Session Library:** The session initialization debug message now uses the correct log type "debug" instead of "info".
```

**Review by michalsn — APPROVED** [review]
Thank you.

In general, we don't enter the PR number in the changelog.

**Comment by element-code** [comment]
Alright, should I squash or do you squash to merge?

**Comment by michalsn** [comment]
We can squash on merge - we just have to wait for one more approval.

**Review by datamweb — APPROVED** [review]
This is a great improvement, it was always bothering me!🤓

**Comment by michalsn** [comment]
@element-code Thank you!

**Comment by mbnl** [comment]
This was a very frustrating situation for me, so I'm glad it got resolved. Also, are there no new features or updates being made to CI4? Has its development been stopped

**Comment by michalsn** [comment]
> Also, are there no new features or updates being made to CI4? Has its development been stopped

@mbnl What do you mean? 

The latest release was a month ago: https://github.com/codeigniter4/CodeIgniter4/releases/tag/v4.5.5

The new features go to the `4.6` branch: https://github.com/codeigniter4/CodeIgniter4/tree/4.6, which is ahead of the `develop` by more than 300 commits.

**Pull request merged**
